src/components/AccountAddresses.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="wallet-identifier">
+        <div v-for="n in 6" class="account" :key="n" :class="{ placeholder: n > addresses.length, 'pop-in': animate }">
+            <Identicon v-if="n <= addresses.length" :address="addresses[n - 1]"></Identicon>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import Identicon from './Identicon.vue';
+
+@Component({components: { Identicon }})
+export default class WalletIdentifier extends Vue {
+    @Prop({ default: () => [], type: Array })
+    public addresses!: string[];
+
+    @Prop({ default: false, type: Boolean })
+    public animate!: boolean;
+}
+</script>
+
+<style scoped>
+    .wallet-identifier {
+        position: relative;
+        --hex-width: 4rem;
+        --hex-height: calc(var(--hex-width) * (24 / 27)); /* from placeholder viewBox="0 0 27 24" */
+        --gap-x: calc(var(--hex-width) / 12);
+        --gap-y: calc(var(--gap-x) * .9);
+        min-height: calc(3 * var(--hex-height) + 2 * var(--gap-y));
+        min-width: calc(2.5 * var(--hex-width) + 2 * var(--gap-x));
+    }
+
+    .account {
+        width: var(--hex-width);
+        height: var(--hex-height);
+        position: absolute;
+    }
+
+    .account .identicon {
+        width: 100%;
+        height: 100%;
+    }
+
+    .placeholder {
+        background: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 27 24" width="100%" height="100%"><path d="M26.6991 10.875L21.0741 1.125C20.6691 0.4275 19.9266 0 19.1241 0H7.87414C7.07164 0 6.32914 0.4275 5.92789 1.125L0.302891 10.875C-0.0983594 11.5725 -0.0983594 12.4275 0.302891 13.125L5.92789 22.875C6.32914 23.5725 7.07164 24 7.87414 24H19.1241C19.9266 24 20.6691 23.5725 21.0704 22.875L26.6954 13.125C27.1004 12.4275 27.1004 11.5725 26.6991 10.875Z"></path></svg>') center no-repeat content-box;
+        opacity: 0.1;
+        color: var(--nq-blue);
```

Comment:
`--nq-blue` does not exist (what you mean is `--nimiq-blue`), which shows that it has no effect here and can be removed. If you want to color the background SVG, you need to add a `fill="%231F2348"` property to the `svg` tag.